### DISCUSSION TITLE
Error while installing

### DISCUSSION THREAD

**Discussion by gputignano** [MahoCommerce/maho#41] [General]
I'm trying to install for the first time Maho script.
First I execute the command:
`composer create-project mahocommerce/maho-starter yourproject`

Then when I execute ./maho install I get the following error:
```
Warning: require_once(C:\wamp64\www\yourproject\vendor\mahocommerce\maho\vendor\autoload.php): Failed to open stream: No such file or directory in C:\wamp64\www\yourproject\vendor\mahocommerce\maho\app\Mage.php on line 48
...
```
What's the problem?

**Comment by fballiano** [comment]
Hi @gputignano, at the moment we only support linux-ish systems like linux and MacOS, I'm not sure if it would work the same in windows+linux-subsystem since I have no way of testing it (but you could use ddev of a docker container or virtualization, not sure which is best)

**Reply by justinbeaty** [reply]
WSL should work, because I think it just a linux VM. I am not sure about WAMP, but in this case I think @gputignano is just missing a composer install?

**Reply by fballiano** [reply]
exactly, I'd expect WSL to work and that would be the only windows-way we'd support

**Reply by fballiano** [reply]
> is just missing a composer install?

composer create-project already does a composer install automatically

**Reply by gputignano** [reply]
I confirm that I have to run a composer install command in yourproject\vendor\mahocommerce\maho.

**Reply by justinbeaty** [reply]
Could it be old composer versions don't do the install after create-project? I don't know...
